examples/SignalingTestApp/SignalingTestApp/ViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ViewController.swift
+//  SignalingTestApp
+//
+//  Created by Gloria Li on 2021-01-11.
+//
+
```

Comment:
Must use the standard iOS license header. It seems this demo app was not generated within the AzureSDK workspace.